plugins/components/health/Health.hpp

```diff
@@ -140,8 +140,12 @@ namespace Components {
                 std::cerr << "Warning: 'maxVal' not found in config. Using default value: 1.0f\n";
                 maxVal = 100;
             }
+
+            std::cout << "currentVal: " << currentVal << " maxVal: " << maxVal << std::endl;
+
             std::unique_ptr<Components::Health> health = engine.newComponent<Components::Health>(static_cast<uint32_t>(currentVal), static_cast<uint32_t>(maxVal));
             engine.getRegistry().componentManager().addComponent<Components::Health>(to, std::move(health));
+            std::cout << std::endl;
```

Comment:
same here

Reply:
Same as before, its for the good continuity of the project.

It prints this kind of debug : 

New entity created with ID: 3
Adding component: "Position" to entity ID: 3
x: 500 y: 0 layer: 1


very usefull for debugging!